doc/modules/gaussian_process.rst

```diff
@@ -570,7 +571,7 @@ It is parameterized by a length-scale parameter :math:`l>0` and a periodicity pa
 The kernel is given by:
 
 .. math::
-   k(x_i, x_j) = \text{exp}\left(-2 \left(\text{sin}(\pi / p * d(x_i, x_j)) / l\right)^2\right)
+   k(x_i, x_j) = \text{exp}\left(- \frac{ 2\sin^2(\pi |x_i - x_j| / p) }{ l^ 2} \right)
```

Comment:
At least in the code, this is actually the euclidean distance, not `|x_i - x_j|`.

Reply:
That's interesting. I did some digging around, comparing different implementations and the formulas they gave. For the mathy formula, most give the absolute difference (George, Cookbook, Tensorflow) but then don't always say what they do to reduce the dimensions when not in a 1-dimensional case. Only Tensorflow mentions that it sums up the squared sines. 
Others (Pyro) just use the difference, also not mentioning how they reduce the dimensions, but summing up the sines in the code. GPFlow says they use the Euclidean distance but the code looks like they're using the difference and then either sum the absolute or squared sines :woman_shrugging: 
The oldest formula I found seems to be from [David MacKay](http://www.inference.org.uk/mackay/gpB.pdf), p. 15, and sums up the squared sines of the simple difference.
Seeing such a wide range of implementations, I'd assume the details don't seem to make a significant difference. I change it back to use the Euclidean distance again.